src/transformers/modeling_outputs.py

```diff
@@ -245,6 +245,41 @@ class BaseModelOutputWithPastAndCrossAttentions(ModelOutput):
     cross_attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class DecisionTransformerOutput(ModelOutput):
```

Comment:
This is specific to Decision Transformers, so it should really go in the `modeling_decision_transformers` file.